ckanext/datajson/plugin.py

```diff
@@ -161,31 +162,23 @@ def validator(self):
 
         return render('datajsonvalidator.html')
 
-    def generate_pdl(self):
-        # DWC this is a hack, as I couldn't get to the request parameters. For whatever reason, the multidict was always empty
-        match = re.match(r"/organization/([-a-z0-9]+)/data.json", request.path)
```

Comment:
Why did we remove this regular expression?

Reply:
Regular expression is not the best way of getting the organization id. The org id is being passed from the route anyway.